src/App.js

```diff
@@ -17,6 +18,7 @@ function App() {
                     <Route path="/register" element={<Register />} />
                     <Route path="/profile" element={<ProfilePage />} />
                     <Route path="/board" element={<GuestHomepage />} />
+                    <Route path="/details/:id" element={<DetailsModal />} />
```

Comment:
The idea was to implement this as a modal, not a separate page, here's some instructions:

- [Tutorial](https://www.digitalocean.com/community/tutorials/react-modal-component)
- Add the modal for employee details to the board page